software/software.mdown

```diff
@@ -21,6 +21,7 @@
 ## Libraries
 
 * **[JMAP-JS](https://github.com/jmapio/jmap-js)** (JavaScript, MIT): a full implementation of the JMAP mail, calendar and contacts model in JavaScript. It supports asynchronous local changes and is tolerant of network failure – actions will update the UI instantly, then synchronise changes back to the server when it can. It also has multi-level undo/redo support. Used by the [demo webmail](https://github.com/jmapio/jmap-demo-webmail).
+* **[jmap-client-ts](https://github.com/OpenPaaS-Suite/jmap-client-ts)** (TypeScript, MIT): a lightweight promise-based API to make JMAP requests.
 * **[jmap-client](https://github.com/linagora/jmap-client)** (JavaScript ES6, MIT): a simple promise-based API to send requests to a JMAP server.
```

Comment:
This client only implements a draft version of the JMAP protocol. While we are at it, we may want to unreference it too, don't you think?